data/lib/src/extensions/uri_extension.dart

```diff
@@ -34,6 +34,10 @@ import 'package:domain/domain.dart';
 
 extension UriExtension on Uri {
   String withServicePath(ServicePath servicePath) {
-    return origin + EndPoint.rootPath + servicePath.path;
+    return origin + Endpoint.rootPath + servicePath.path;
+  }
+
+  String withDownloadPath(ServicePath servicePath, DocumentId documentId) {
+    return withServicePath(servicePath) + '/${documentId.uuid}${Endpoint.download}';
   }
```

Comment:
I remembered I comment for this method. It is not generic enough for `SharedSpaceDocumentId`, `ReceivedShareId`, ...

Reply:
You are right, but I tried to keep as simple as possible first that's why. I'll look to make it generic